readme.md

```diff
@@ -1,6 +1,8 @@
 > This is a list of must-watch videos devoted to JavaScript
 
 ## 2015
+1. [Andre Staltz: What if the user was a function?](https://www.youtube.com/watch?v=1zj7M1LnJV4) [32:19]
+1. [Sufian Rhazi: Transform your codebase](https://www.youtube.com/watch?v=X8WgkMbEAVQ) [33:49]
```

Comment:
Here's another [talk](https://www.youtube.com/watch?v=rKuNbEwoQfQ) related to the similar topic from the author of Babel. Take a look, let me know what you think.